config/initializers/devise.rb

```diff
@@ -236,6 +237,7 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :google_oauth2, "354458150058-imo3ck7hnq1vh3hupfj61enf4kols6q2.apps.googleusercontent.com", "6pxDYYQVuu879shj8c-NslFl", { access_type: "offline", approval_prompt: "", :client_options => {:ssl => {:verify => false}} }
```

Comment:
Line is too long. [234/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.